opennms-webapp-rest/src/main/java/org/opennms/web/rest/v2/README.adoc

```diff
@@ -0,0 +1,479 @@
+== OpenNMS RESTv2 Interface
```

Comment:
I'd also like to see this in the developers guide as this is at the moment the place where the REST API is documented. Any new documentation - finished or unfinished - should go there.